lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -66,6 +66,7 @@ my %URL_SPECIALIZATIONS = (
     'DRAM'                => qr{^https?://(?:www\.)?dramonline\.org/}i,
     'DynamicRangeDB'      => qr{^https?://dr\.loudness-war\.info/}i,
     'Encyclopedisque'     => qr{^https?://(?:www\.)?encyclopedisque\.fr/}i,
+    'eonkyo'              => qr{^https?://(?:www\.)?e-onkyo\.com/}i,
```

Comment:
Just to follow the current convention, the Perl package name should rather be capitalized:
```suggestion
    'EOnkyo'              => qr{^https?://(?:www\.)?e-onkyo\.com/}i,
```

Reply:
Resolved in [666952b](https://github.com/metabrainz/musicbrainz-server/pull/3328/commits/666952b9614532302c2914ed7fb51e96d0d79d12)!